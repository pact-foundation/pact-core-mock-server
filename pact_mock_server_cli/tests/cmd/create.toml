bin.name = "pact-mock-server"
args = "create --help"
