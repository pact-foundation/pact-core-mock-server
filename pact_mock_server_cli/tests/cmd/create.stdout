Creates a new mock server from a pact file

Usage: pact-mock-server create [OPTIONS] --file <file>

Options:
  -f, --file <file>                    the pact file to define the mock server
      --help                           Print help and exit
  -c, --cors-preflight                 Handle CORS pre-flight requests
  -v, --version                        Print version information and exit
  -p, --port <port>                    port the master mock server runs on (defaults to 8080)
      --specification <specification>  The Pact specification version to use (defaults to V4)
  -h, --host <host>                    hostname the master mock server runs on (defaults to localhost)
      --tls                            Enable TLS with the mock server (will use a self-signed certificate)
  -l, --loglevel <loglevel>            Log level for mock servers to write to the log file (defaults to info) [possible values: error, warn, info, debug, trace, none]
      --no-term-log                    Turns off using terminal ANSI escape codes
      --no-file-log                    Do not log to an output file

